staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_volume_map.go

```diff
@@ -69,6 +71,16 @@ func (vsphereVolume *VsphereVolumeMap) CheckForVolume(path string) (k8stypes.Nod
 	return "", false
 }
 
+func (vsphereVolume *VsphereVolumeMap) CheckForNode(nodeName k8stypes.NodeName) bool {
```

Comment:
Could you document the meaning of "known" node here or somewhere nearby? Is it different from "nodes in this cluster"?

Reply:
I have renamed the variable and clarified the intent of the check.